OpenRA.Mods.Common/Traits/Render/WithMoveAnimation.cs

```diff
@@ -46,17 +49,40 @@ public WithMoveAnimation(ActorInitializer init, WithMoveAnimationInfo info)
 			wsb = init.Self.TraitsImplementing<WithSpriteBody>().Single(w => w.Info.Name == Info.Body);
 		}
 
-		void ITick.Tick(Actor self)
+		void INotifyMoving.StartedMoving(Actor self)
 		{
-			if (IsTraitDisabled || wsb.IsTraitDisabled)
-				return;
+			if (!IsTraitDisabled && !wsb.IsTraitDisabled)
+				wsb.PlayCustomAnimationRepeating(self, Info.MoveSequence);
```

Comment:
Likewise here - it would be more robust if we could have a single inner method that plays/cancels, and is then called with whatever args are needed to distinguish the cases from the interface/enabled